specification/ai/DocumentIntelligence/tspconfig.yaml

```
@@ -18,11 +18,11 @@ options:
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/DocumentIntelligence.json"
   # Uncomment this line and add "@azure-tools/typespec-python" to your package.json to generate Python code
   "@azure-tools/typespec-python":
-    "basic-setup-py": true
-    "package-version": 1.0.0b1
-    "package-dir": "azure-ai-documentintelligence"
-    "package-name": "azure-ai-documentintelligence"
-    # "output-path":
+    package-pprint-name: "\"Azure AI Document Intelligence\""
```

Nit the emitter fixed the issue that required the double quotes:
```suggestion
    package-pprint-name: "Azure AI Document Intelligence"
```